### PR TITLE
Add VTOL modes to command list and re-org into cmd types

### PR DESCRIPTION
**Pull request by Hwurzburg**
first phase....added VTOL plane commands, unified grouping and ordering into NAV/Special/Conditional/DO for all vehicles....after this merge, will work on verifying Mission Planner actually supports all commands (apparently several plane one are not), then create individual NAV pages for each vehicle (like copter already has) with a common page for conditionals and do's linked via a landing page....and delete this one...too many vehicle differences on too many commands...

### CODE REVIEW

File: common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst [hunk 1]
```diff
@@ -924,6 +750,79 @@ using an airspeed sensor.
...
+Takeoff while in VTOL mode.
+
+
+QuadPlane
+~~~~~~~~~
+
+The vehicle will climb straight up from it’s current location to the
+specified altitude. If the mission is begun while the vehicle is already
+flying, the vehicle will climb straight up to the specified altitude, if
+the vehicle is already above the altitude the command will be ignored
```
Comment (tridge) [inline common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst]: The default behaviour is that the altitude is relative to the altitude at the time the command starts. If Q_OPTIONS bit 3 is set then the altitude instead uses the frame type of the command.

File: common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst [hunk 2]
```diff
@@ -1740,6 +1628,74 @@ description here?.
    <td>Long</td>
    <td>Longitude</td>
    </tr>
+   <td><strong>param7</strong></td>
+   <td>Alt</td>
+   <td>Altitude to target for the landing.Unless you are landing at a location different than home, this should be zero</td>
```
Comment (tridge) [inline common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst]: space missing

Comment (Hwurzburg) [reply common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst]: fixed

**Comment by Hwurzburg** [comment]
@tridge thanks for review....merging

**Pull request merged**